specifications/verifiable.presentation.protocol.md

```diff
@@ -139,6 +140,10 @@ A scope is a string value in the form:
 
 The `[alias]` value MAY be implementation-specific.
 
+<aside class="example" title="Requesting a Token with Scopes">
```

Comment:
there's an issue to remove this entire section. Also (as stated) this should be added in the base protocol.